src/Storage/Entity/Entity.php

```diff
@@ -73,8 +77,36 @@ public function setId($id)
         $this->id = $id;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function __toString()
     {
         return (string) $this->getId();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function toArray()
+    {
+        $data = [];
+        foreach ($this as $k => $v) {
```

Comment:
… and believe it or not, the `getUsers()` thing was BC, no idea if what was requiring it to maintain the BC is fixed, and the class, globals, and a tonne else, is going in `master` already or soon.